src/pathfinding_service/api.py

```diff
@@ -33,7 +33,7 @@
 from raiden_libs.types import Address, TokenNetworkAddress
 
 log = structlog.get_logger(__name__)
-
+g.last_request = []
```

Comment:
The `g` object is only usable after the flask application has been initialized, because it belongs to the application context. Instead use
```python
if 'last_request' no in g:
    g.last_request = []
```
where using that attribute,

Reply:
done